### PR TITLE
OSP-84: Update the open recruitment types to include possible studies 

### PR DESCRIPTION
**Pull request by LiekeSchoenmaker**
I have discussed the different recruitment types to include possible studies for interpretation. For this reason a small update is necessary to inlcude the following recruitment types: 

Recruiting
Not yet recruiting
Available
Approved for marketing
not_yet_recruiting --> It is for me not clear why we should interpret this? 

see ticket https://hartwigmedical.atlassian.net/browse/OSP-84

### CODE REVIEW

**Review by pauldwolfe — CHANGES_REQUESTED** [review]
We still require a ticket even for small changes.

File: algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java [hunk 1]
```diff
@@ -30,11 +31,9 @@ class ActionableTrialFactory implements ActionableEntryFactory {
 
     private static final Set<String> AGE_GROUPS_TO_INCLUDE = Sets.newHashSet();
 
-
     static {
         POTENTIALLY_OPEN_RECRUITMENT_TYPES.add("recruiting");
```
Comment (pauldwolfe) [inline algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java:35]: you can also create a set with `Set.of()` and remove the need for a mutable set and the static block

File: algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java [hunk 2]
```diff
@@ -163,4 +162,9 @@ static boolean hasAgeGroupToInclude(@NotNull List<String> ageGroups) {
         }
         return false;
     }
+
+    @VisibleForTesting
+    static boolean hasPotentiallyOpenRequirementToInclude(@Nullable String locationStatus) {
```
Comment (pauldwolfe) [inline algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java:167]: Is `locationStatus` a really descriptive name? In one place the method is called with `trial.recruitment()`, and the values look more like a recruitment status than the status of a location.

**Review by pauldwolfe — APPROVED** [review]
Couple small ideas to improve, but approved. 

When you merge, please include the ticket in the title of the commit.

File: algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java (hunk 1 above)
Comment (LiekeSchoenmaker) [reply algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java:35]: done

File: algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java (hunk 2 above)
Comment (LiekeSchoenmaker) [reply algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java:167]: indeed, I have update it to recruitmentStatus

**Review by cbruel — APPROVED** [review]

**Pull request merged**